packages/runtime/test/dataViews/PeerAttributeDVO.test.ts

```diff
@@ -0,0 +1,214 @@
+import { AbstractIntegerJSON, AbstractStringJSON, IdentityAttributeJSON } from "@nmshd/content";
+import { PeerAttributeDVO } from "../../src";
+import { ensureActiveRelationship, executeFullCreateAndShareRepositoryAttributeFlow, RuntimeServiceProvider, TestRuntimeServices } from "../lib";
+
+const serviceProvider = new RuntimeServiceProvider();
+let services1: TestRuntimeServices;
+let services2: TestRuntimeServices;
+
+beforeAll(async () => {
+    const runtimeServices = await serviceProvider.launch(2, {
+        enableRequestModule: true,
+        enableDeciderModule: true,
+        enableNotificationModule: true
+    });
+    services1 = runtimeServices[0];
+    services2 = runtimeServices[1];
+
+    await ensureActiveRelationship(services1.transport, services2.transport);
+}, 30000);
+
+afterAll(() => serviceProvider.stop());
+
+describe("PeerAttributeDVO", () => {
+    test("check the BirthYear", async () => {
+        const sOwnSharedIdentityAttribute = await executeFullCreateAndShareRepositoryAttributeFlow(services2, services1, {
```

Comment:
I know what you mean by "sOwnSharedIdentityAttribute" and "rPeerSharedIdentityAttribute". I think for people who are not so familiar with the naming conventions, it can be difficult to understand what "s" and "r" is referring to. Therefore, perhaps use "sender" and "recipient" here instead. 

If this is too long for you, maybe instead of "sOwnSharedIdentityAttribute" it could also be written "s2OwnSharedIdentityAttribute" and instead of "rPeerSharedIdentityAttribute" it could also be written "s1PeerSharedIdentityAttribute" to refer to "services2" or "services1", respectively.

Same for other tests.
(But if you do not like any of the suggested options better, it is okay for me to leave it as it is for now. 😊)

Reply:
I used "s" and "r" prefixes, since they are used in a lot of different tests in different libs throughout our monorepo. "s1" and "s2" aren't more speaking to me.